spec/models/entity_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Entity, type: :model do
+  describe 'associations' do
+    it 'belongs to a user' do
+      association = Entity.reflect_on_association(:user)
+      expect(association.macro).to eq(:belongs_to)
+    end
```

Comment:
- [ ] Good job adding models and controller specs, As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=Testing%20requirements-,Create%20unit%20and%20integration%20tests%20for%20all%20the%20most%20important%20components%20of%20your%20RoR%20application.,-Technical%20requirements), Please add integration specs for all the most important components of your RoR application too.